builder/builder-next/executor_nolinux.go

```diff
@@ -0,0 +1,33 @@
+//go:build !linux
```

Comment:
So, I'm actually wondering why we're initialising `BuildKit` at all on Windows, if we don't support it yet; i.e., we could skip all of this, and wouldn't have to stub anything; https://github.com/moby/moby/blob/0d9da7367dd79e6e5c46ef4a37ca0fbe79d2bf8f/cmd/dockerd/daemon.go#L405-L434